graylog2-web-interface/src/components/lookup-tables/DataAdapterTableEntry.jsx

```diff
@@ -22,10 +29,18 @@ const DataAdapterTableEntry = React.createClass({
   },
 
   render() {
+    const errorOverlay = !this.props.error ? null : (
+      <Popover id="popover-table-error" title="Lookup Table problem" style={{ maxWidth: 400 }}>
```

Comment:
Somehow related: we have been trying to avoid inline styles for a while. Could you put them in a stylesheet so we can reuse them? In this case `LUTableEntry` would also benefit from it.